### PR TITLE
fix(TBD-10541): Upgrade jackson dependencies to 2.9.10.4 for MapR 6.0.1

### PR DESCRIPTION
**Pull request by mtuhai**
**Please check if the PR fulfills these requirements**

- [X] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [X] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [X] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [X] Docs have been added / updated (for bug fixes / features) ?
- [X] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)
TBD-10541

**What is the new behavior?**
Fixed

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.mapr601/plugin.xml [hunk 1]
```diff
@@ -381,21 +381,21 @@
         </libraryNeeded> 
         <libraryNeeded 
             context="plugin:org.talend.libraries.hadoop.mapr.6.0.1" 
-            id="jackson-databind-2.9.5.jar" 
-            name="jackson-databind-2.9.5.jar"
-			mvn_uri="mvn:com.fasterxml.jackson.core/jackson-databind/2.9.5">
+            id="jackson-databind-2.9.10.jar"
```
Comment (CarlosTalend) [inline main/plugins/org.talend.hadoop.distribution.mapr601/plugin.xml:384]: Could we please use the 2.9.10.4 version instead for jackson-databind?

File: main/plugins/org.talend.hadoop.distribution.mapr601/plugin.xml [hunk 2]
```diff
@@ -218,10 +218,10 @@
...
-            name="hive-jdbc-1.2.0-mapr-1803-spark.jar"
-            mvn_uri="mvn:org.talend.libraries/hive-jdbc-1.2.0-mapr-1803-spark/6.0.0" >
-        </libraryNeeded>
+            id="hive-jdbc-1.2.0-mapr-1912-spark.jar"
+            name="hive-jdbc-1.2.0-mapr-1912-spark.jar"
+            mvn_uri="mvn:org.talend.libraries/hive-jdbc-1.2.0-mapr-1912-spark/6.0.0" >
+       </libraryNeeded>
        <libraryNeeded
             context="plugin:org.talend.libraries.hadoop.mapr.6.0.1"
             id="hive-service-2.1.1-mapr-1803.jar"
```
Comment (CarlosTalend) [inline main/plugins/org.talend.hadoop.distribution.mapr601/plugin.xml:227]: Why are there some "1803 without the r2" dependencies that weren't changed to "1912", while others where? (this line is just an example)

Comment (mtuhai) [reply main/plugins/org.talend.hadoop.distribution.mapr601/plugin.xml:227]: This is what comes with MapR update. Some libraries changed version to 1912-r4, some changed to 1912 and some stays unmodified

File: main/plugins/org.talend.hadoop.distribution.mapr601/plugin.xml (hunk 1 above)
Comment (mtuhai) [reply main/plugins/org.talend.hadoop.distribution.mapr601/plugin.xml:384]: But MapR update uses jackson-databind-2.9.10

Comment (CarlosTalend) [reply main/plugins/org.talend.hadoop.distribution.mapr601/plugin.xml:384]: OK! They told me by email differently, but the truth is held by their patch. Let's roll with this modification then.

File: main/plugins/org.talend.hadoop.distribution.mapr601/plugin.xml (hunk 2 above)
Comment (CarlosTalend) [reply main/plugins/org.talend.hadoop.distribution.mapr601/plugin.xml:227]: Thanks!

**Comment by CarlosTalend** [comment]
#1431 addresses this issue

**Pull request closed without merging**